index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+<h1 class="favourite">Hello Microverse!.</h1>    
+</body>
```

Comment:
- [ ] [OPTIONAL] I would suggest following an indentation convention because it is considered a good practice and also it will make your code more readable :raised_hands: